sdk/textanalytics/ai-text-analytics/src/generated/models/index.ts

```diff
@@ -466,6 +489,14 @@ export interface SentenceSentiment {
    * The sentiment confidence score between 0 and 1 for the sentence for all classes.
    */
   confidenceScores: SentimentConfidenceScores;
+  /**
+   * The sentence offset from the start of the document.
```

Comment:
encoding needed here too

Reply:
Same comments about including `text` to the docstring.
same for length

Reply:
these are all in the `generated` directory which we do not ever manually change.